pages/resources/user.mdx

```diff
@@ -450,6 +450,92 @@ How a user's name gets displayed, such as font, colors, gradient, glow.
 | 5     | POP      | Colored dropshadow                  |
 | 6     | GLOW     | Alternate gradient style            |
 
+## Game widgets
+
+#### Game Widgets Terminology Specification
+This section may contain terminology which can easily get mixed up. This list specifies their definition.
+
+  - Game Widget: A game widget is a single widget list (e.g. favourite games, games in rotation) containing Game objects.
+  - Game/Game Object: A "Game" is in this case referring to a game within a game widget list, containing an optional comment and tags.
+  - Detailed Game Widget: A detailed game widget is a game widget where the games in the list can display tags and a comment.
+  - Application Game Widget: An application game widget is a special kind of game widget which displays special data/statistics specific to a supported game.
+
```

Comment:
```suggestion
```
unfortunately this section just makes it more confusing. game widgets aren't really in need of a terminology section cuz you can just follow the logical path of the structures